Makefile

```diff
@@ -16,7 +16,7 @@ install: os_migrate-os_migrate-latest.tar.gz
 		source /root/venv/bin/activate; \
 	fi; \
 	cd releases; \
-	ansible-galaxy collection install --force os_migrate-os_migrate-latest.tar.gz
+	ansible-galaxy collection install $(OS_MIGRATE_INSTALL_ARGS) --force os_migrate-os_migrate-latest.tar.gz
```

Comment:
Will split this out into its own commit. It allows passing `--no-deps` to the collection install, which speeds up commands like `./toolbox/run make test-unit` by a lot.

Reply:
Nice addition